src/main/java/com/fdmgroup/forex/enums/OrderStatus.java

```diff
@@ -0,0 +1,13 @@
+package com.fdmgroup.forex.enums;
+
+/**
+ * Denotes the state of the order as follows:
+ * ACTIVE: Order is live and open to matches
+ * CANCELLED: Order cancelled by the user
```

Comment:
Order can be CANCELLED on InsufficientFundException too, when no partial trades ever happened

Reply:
[a687131](https://github.com/forex-agile/backend/pull/8/commits/a687131dfe403beabca4640ce17d97442725c208)